bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/NetatmoHandlerFactory.java

```diff
@@ -90,7 +90,7 @@ public boolean supportsThingType(ThingTypeUID thingTypeUID) {
     }
 
     private @Nullable BaseThingHandler buildThing(Bridge bridge, ModuleType moduleType) {
-        List<AbstractChannelHelper> helpers = new ArrayList<>();
+        List<@Nullable AbstractChannelHelper> helpers = new ArrayList<>();
```

Comment:
I was also wondering and tried some things but nothing I did made the compiler happy. Perhaps additional annotations at other places would help, but I wanted a "quick" fix so that I could build.